lib/twurl/cli.rb

```diff
@@ -95,13 +93,15 @@ def parse_options(args)
         arguments                 = option_parser.parse!(args)
         Twurl.options.command     = extract_command!(arguments)
         Twurl.options.path        = extract_path!(arguments)
-
+        Twurl.options.subcommands = arguments
+        
         if Twurl.options.command == DEFAULT_COMMAND and Twurl.options.path.nil?
-          CLI.puts option_parser
-          raise NoPathFound, "No path found"
+          if Twurl.options.args.first.nil? or Twurl.options.subcommands.nil?
```

Comment:
```suggestion
          if Twurl.options.command == DEFAULT_COMMAND and Twurl.options.args.empty?
```

> In that case, I added some extra conditions. We can only raise it when the program is run without arguments or if it is not an alias.

@flaviomartins The added later condition`Twurl.options.subcommands.nil?` will never be true because it should be an empty list in case if there are not subcommands. So this should be `.empty?`.

But, it still causes a problem. Let's say if you execute this command: `$ twurl accounts` then the output of `Twurl.options` would be something like this:
> #<Twurl::Options args=["accounts"], trace=false, data={}, headers={}, upload={"file"=>[]}, command="accounts", path=nil, subcommands=[]>

In this case, the added condition would be true and shows usage output even though it is a valid syntax.

Reply:
@flaviomartins Let me know if you have concerns or you can commit my suggestion directly. Once this PR is merged, we're going to release a new version...!